universal-login-react/test/core/services/DeleteAccountService.ts

```diff
@@ -0,0 +1,51 @@
+import chai, {expect} from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import {deleteAccount} from '../../../src/core/services/DeleteAccountService';
+import {DeployedWallet, WalletService} from '@universal-login/sdk';
+import {Wallet} from 'ethers';
+import {getWallets, createMockProvider} from 'ethereum-waffle';
+import {setupDeployedWallet} from '../../helpers/setupDeploymentWallet';
+import Relayer from '@universal-login/relayer';
```

Comment:
I believe this is `RelayerUnderTest`

Reply:
RelayerUnderTest doesn't create sdk and deployedWallet

Reply:
A relayer that is returned in `setupSdk` is RelayerUnderTest type.

Reply:
Ok, sorry, I understood your comment wrongly

Reply:
ok  👍